### PR TITLE
Updated misc devel tools.

### PR DESCRIPTION
**Pull request by stefanosoffia**
Updated some developer tools to their latest versions.

### CODE REVIEW

File: var/spack/repos/builtin/packages/valgrind/package.py [hunk 1]
```diff
@@ -38,8 +38,9 @@ class Valgrind(AutotoolsPackage):
 
     """
     homepage = "http://valgrind.org/"
-    url = "http://valgrind.org/downloads/valgrind-3.11.0.tar.bz2"
+    url = "ftp://sourceware.org/pub/valgrind/valgrind-3.13.0.tar.bz2"
```
Comment (healther) [inline var/spack/repos/builtin/packages/valgrind/package.py]: Why did you change the protocol? For our system for example it is non-trivial to download from ftp sources

Comment (stefanosoffia) [reply var/spack/repos/builtin/packages/valgrind/package.py]: That one seems to be now the [official download location](http://valgrind.org/downloads/current.html#current) for valgrind. Specifically, the new release `3.13.0` is not available with the old link. Tough, if the problem is `ftp`, the new link has also an https. Let me update it to `https`.

Comment (healther) [reply var/spack/repos/builtin/packages/valgrind/package.py]: Thanks `https` works out of the box :+1:
